Makefile

```diff
@@ -183,11 +175,7 @@ toitpkg: $(TOITPKG_BIN)
 
 TOITPKG_VERSION := v0.0.0-20211126161923-c00da039da00
 $(TOITPKG_BIN):
-ifeq ($(GO_USE_INSTALL), 1)
 	GOBIN="$(CURDIR)"/$(dir $@) go install github.com/toitlang/tpkg/cmd/toitpkg@$(TOITPKG_VERSION)
```

Comment:
Should we check the go version?

Reply:
That's what the old code was trying to do, and it failed on Windows.
If the version isn't new enough there is an error, which can easily be googled. I can try to find a better way to check the version, but it would take time.
It is true, that Go 1.16 is relatively new, though (Feb 2021).